.github/workflows/bump_packages.yml

```
@@ -0,0 +1,95 @@
+name: Update packages
+on:
+  workflow_dispatch:
+env:
+  GIT_AUTHOR_NAME: Foreman Packaging Automation
+  GIT_AUTHOR_EMAIL: packaging@theforeman.org
+
+jobs:
+  rpm_list:
+    name: 'Gather RPMs'
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-list.outputs.matrix }}
+    steps:
+    - name: Checkout RPM
+      uses: actions/checkout@v2
+      with:
+        ref: rpm/develop
+    - name: Set the list
+      id: set-list
```

```suggestion
      id: set_list
```

I always get goose bumps when I see dashes in things that equally could be computations ;)